earth_enterprise/src/installer/install_server.sh

```diff
@@ -268,112 +274,125 @@ configure_publish_root()
 {
   # Update PUBLISHER_ROOT if geserver already installed
   local STREAM_SPACE="$GEHTTPD_CONF/stream_space"
-  if [ -e $STREAM_SPACE ]; then
-    PUBLISHER_ROOT=`cat $STREAM_SPACE |cut -d" " -f3 |sed 's/.\{13\}$//'`
+  if [ -e "$STREAM_SPACE" ]; then
+    PUBLISHER_ROOT=`cat $STREAM_SPACE | cut -d" " -f3 | sed 's/.\{13\}$//'`
   fi
 }
 
+backup_pgsql_data()
+{
+  # Make the temporary data backup directory
+  mkdir -p "$BACKUP_DATA_DIR"
+  chown "$GEPGUSER_NAME $BACKUP_DATA_DIR"
```

Comment:
missing ". `chown "$GEPGUSER_NAME $BACKUP_DATA_DIR"` needs to be changed to: `chown "$GEPGUSER_NAME" "$BACKUP_DATA_DIR"`

The command as it is fails to give permission to that user and PostgreSQL dumps are not created in consequence.